src/lib/crypto/kimchi_backend/common/plonk_dlog_proof.ml

```diff
@@ -270,7 +268,12 @@ module Make (Inputs : Inputs_intf) = struct
     ; foreign_field_mul_selector
     ; xor_selector
     ; rot_selector
-    ; lookup = Option.map ~f:lookup_eval_of_backend lookup
+    ; lookup_aggregation
+    ; lookup_table
+    ; lookup_sorted =
+        Array.init 5 ~f:(fun i ->
```

Comment:
`5` seems to be a little bit magic here. Could there be an explanation in the form of comments?

Reply:
Sure. Max number of columns in the sorted lookup argument